cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -330,4 +331,73 @@ describe('Top toolbar tests.', function() {
 			.should('exist');
 	});
 
+	it('Save.', function() {
+		cy.get('#tb_editbar_item_bold')
+			.click();
+
+		cy.get('#tb_editbar_item_save')
+			.click();
+
+		helper.beforeAll(testFileName, 'writer', true);
+
+		helper.reselectAllText();
+
+		cy.get('#copy-paste-container p b')
+			.should('exist');
+
+	});
+
+	it('Print', function() {
+		// A new window should be opened with the PDF.
+		cy.window()
+			.then(function(win) {
+				cy.stub(win, 'open');
+			});
+
+		cy.get('#tb_editbar_item_print')
+		    .click();
+
+		cy.window().its('open').should('be.called');
+	});
+	
+	it('Apply Undo/Redo.', function() {
+		cy.get('#tb_editbar_item_italic')
+			.click();
+
+		helper.selectAllText(false);
+
+		cy.get('#copy-paste-container p i')
+			.should('exist');
+		helper.selectAllText(false);
+
+		//Undo
+		cy.get('#tb_editbar_item_undo')
+			.click();
+
+		helper.selectAllText(false);
+
+		cy.get('#copy-paste-container p i')
+			.should('not.exist');
+
+		//Redo
+		cy.get('#tb_editbar_item_redo')
+			.click();
+
+		helper.selectAllText(false);
+
+		cy.get('#copy-paste-container p i')
+			.should('exist');
+	});
+
+	it('show sidebar.', function() {
```

Comment:
Sorry, I was not clear here. It was good to have hide and show sidebar in the same test. Under this: "We use those helper methods in other tests, but it's good to have show/hide sidebar test cases covered here alone." I meant, that in other places we might use these helper methods just as a tool to test a different feature. In this test suite, we test showing/hiding sidebar directly and don't do anything else. That's all I meant. Your original idea to cover hiding and showing sidebar in the same test case was good, so we can avoid code duplication. Can you merge these two tests into one and use the helper methods, please?